tests/flow_test.go

```diff
@@ -2032,3 +2032,63 @@ func TestOvsMirror(t *testing.T) {
 
 	RunTest(t, test)
 }
+
+func TestSFlowCapture(t *testing.T) {
+	test := &Test{
+		setupCmds: []helper.Cmd{
+			{"ovs-vsctl add-br br-sfct", true},
+
+			{"ovs-vsctl add-port br-sfct sfct-intf1 -- set interface sfct-intf1 type=internal", true},
+			{"ip netns add sfct-vm1", true},
+			{"ip link set sfct-intf1 netns sfct-vm1", true},
+			{"ip netns exec sfct-vm1 ip address add 169.254.29.11/24 dev sfct-intf1", true},
+			{"ip netns exec sfct-vm1 ip link set sfct-intf1 up", true},
+
+			{"ovs-vsctl add-port br-sfct sfct-intf2 -- set interface sfct-intf2 type=internal", true},
+			{"ip netns add sfct-vm2", true},
+			{"ip link set sfct-intf2 netns sfct-vm2", true},
+			{"ip netns exec sfct-vm2 ip address add 169.254.29.12/24 dev sfct-intf2", true},
+			{"ip netns exec sfct-vm2 ip link set sfct-intf2 up", true},
+
+			{"ovs-vsctl --id=@sflow create sflow agent=lo target=\"127.0.0.1:6343\" header=128 sampling=512 polling=10 -- set bridge br-sfct sflow=@sflow", true},
```

Comment:
please use polling=0 otherwise ovs generates packets not recognized by gopacket, by wireshark neither btw. Use sampling=1 to be sure to catch something.